src/components/common/MapDateSelect/index.tsx

```diff
@@ -0,0 +1,78 @@
+import DateSlider from './DateSlider';
+import { Box, Typography, useTheme } from '@mui/material';
+import { useMemo, useState } from 'react';
+import Card from 'src/components/common/Card';
+import { useLocalization } from 'src/providers';
+
+type MapDateSelectProps = {
+  dates: string[]; // date strings in the format 'YYYY-MM-DD'
+  onChange: (newDate: string) => void;
+};
+
+export default function MapDateSelect({ dates, onChange }: MapDateSelectProps): JSX.Element {
+  const theme = useTheme();
+  const locale = useLocalization().selectedLocale;
+
+  const earliestDate = useMemo(() => {
+    return dates.reduce((prev, curr) => {
+      if (Date.parse(prev) < Date.parse(curr)) {
+        return prev;
+      }
+      return curr;
+    });
+  }, [dates]);
+
+  const latestDate = useMemo(() => {
+    return dates.reduce((prev, curr) => {
+      if (Date.parse(prev) > Date.parse(curr)) {
+        return prev;
+      }
+      return curr;
+    });
+  }, [dates]);
+
+  const latestDateIndex = useMemo(() => {
+    return dates.findIndex((d) => d === latestDate);
+  }, [dates, latestDate]);
+
+  const [selectedDate, setSelectedDate] = useState(latestDate);
+
+  const handleChange = (newDate: string) => {
+    setSelectedDate(newDate);
+    onChange(newDate);
+  };
+
+  const getDateString = (date: string) => {
+    return new Intl.DateTimeFormat(locale, { month: 'short', year: 'numeric', timeZone: 'UTC' }).format(new Date(date));
```

Comment:
Add a note/TODO to re-visit planting site's timezone here? Example: if date string is UTC and planting site time zone is hawaii, we have to use hawaii time zone in the options.
Remove 'UTC' timezone for now since that's the default anyways?

Reply:
If `timeZone: UTC` is left out, then the formatter localizes to the browser timezone, so it needs to stay in.

I'm not sure that this needs to be revisited. If the calculations internal to this component are all in UTC, and it accepts and returns datestrings only, then the timezone is essentially factored out.

Reply:
> If timeZone: UTC is left out, then the formatter localizes to the browser timezone, so it needs to stay in.
Thanks for clarifying the browser timezone. My bad.

> I'm not sure that this needs to be revisited. If the calculations internal to this component are all in UTC, and it accepts and returns datestrings only, then the timezone is essentially factored out.

The time zone is only a display option, not for calculations. If the dates are in UTC but a planting site is in another time zone, we should be using that time zone for generating the display string. Unless I'm missing something that you are trying to convey.

Reply:
Sorry that I'm not clear. What I was trying to say was that the component receives through props and returns through its callback dates that are in 'yyyy-mm-dd' format, only. It only ever displays a 'Month YYYY' string, so it shouldn't need to know about the timezone. However, if I'm understanding what you're saying, then the date that is sent to this component might be in UTC rather than localized to the planting site.

Since, as you say, we don't exactly know how the dates are getting produced by observations and sent to this component (UTC? local to planting site?) I will add a TODO to revisit. Apologies for the back and forth on this.

Reply:
All good. Thanks, yes, only because we don't know if the date will be in UTC or already accounts for the planting site.
Totally agree, if it's the latter.. no change needed.
Just trying to avoid a corner case (showing next month or previous month where a time zone is before or after UTC) at the beginning/end of months.